prime-router/metadata/fhir_mapping/hl7/datatypes/EIP/Identifier-PlacerAssignedIdentifier.yml

```diff
@@ -0,0 +1,113 @@
+value:
+  type: STRING
+  valueOf: EIP.1.1
+  expressionType: HL7Spec
+
+type:
```

Comment:
It looks like this can be hardcoded?  The mapping specifies that this should always be the same value right?

Reply:
Yup it looks like I can remove the variables and just hardcode the strings